### PR TITLE
initial plan for week 3

### PR DESCRIPTION
**Pull request by jinpark**
@hassanabidpk @sujinleeme @RachellCalhoun can you take a look?

### CODE REVIEW

File: lesson_3/readme.md [hunk 1]
```diff
@@ -1,2 +1,42 @@
-# Order:
-1. `word_chain.md`
+# Django Girls Code Camp Week 3
+
+Requirements:
+
+- Finish CodeCademy
```
Comment (hassanabidpk) [inline lesson_3/readme.md:5]: How can we know their status of CodeAcademy ? Have we asked them to show us completion percentage ?

File: lesson_3/readme.md [hunk 2]
```diff
@@ -1,2 +1,42 @@
...
+- Attempt Exercises
+
```
Comment (hassanabidpk) [inline lesson_3/readme.md:7]: Does someone check their code?

File: lesson_3/readme.md [hunk 3]
```diff
@@ -1,2 +1,42 @@
...
+---
+
+| Time  | Content |
+| ------------- | ------------- |
+| 1:10-1:30  | Overview of the day (Rachell/DY)  |
```
Comment (hassanabidpk) [inline lesson_3/readme.md:12]: Maybe giveaway Intel boards to those who come at 1.00 pm sharp! ?

File: lesson_3/readme.md (hunk 1 above)
Comment (jinpark) [inline lesson_3/readme.md:5]: No, I dont think we should. They should do it themselves :P

File: lesson_3/readme.md [hunk 4]
```diff
@@ -1,2 +1,42 @@
...
+| 4:15-4:30  | Talk about dxd (Jangho)  |
+| 4:30-5:45  | Exercises  |
+| 5:45  | Finish up and closing remarks  |
+
+Students:
+---
+
+1. Exercises
+
+    - `word_chain.md`
```
Comment (hassanabidpk) [inline lesson_3/readme.md]: Somebody will translate description to Korean? It seems very long

File: lesson_3/readme.md (hunk 3 above)
Comment (jinpark) [inline lesson_3/readme.md:12]: I thought we decided to not give them away that day if we're not going to show them how to use them that day

File: lesson_3/readme.md (hunk 2 above)
Comment (jinpark) [inline lesson_3/readme.md:7]: Hmm I dont have that setup but I don't know if we should?

File: lesson_3/readme.md (hunk 4 above)
Comment (jinpark) [inline lesson_3/readme.md]: Its pretty long. Thats the only way that the students will finish it I think :p

File: lesson_3/readme.md (hunk 3 above)
Comment (hassanabidpk) [inline lesson_3/readme.md:12]: ok

**Comment by hassanabidpk** [comment]
Looks fine

**Pull request merged**